src/app/(BottomTabNavigation)/AllCases/QRCodeScanner/index.tsx

```diff
@@ -22,9 +29,29 @@ function QRCodeScannerScreen() {
   const [scanned, setScanned] = useState<boolean>(false);
   const [validIds, setValidIds] = useState<CaseUid[]>([]);
   const [userIds, setUserIds] = useState<CaseUid[]>([]);
-  const [toast, setToast] = useState<string>();
+  const [userCase, setUserCase] = useState<Case>();
   const navigation = useNavigation();
 
+  const toastConfig: ToastConfig = {
+    success: (props: any) => <BaseToast />,
+
+    error: (props: any) => (
+      <ErrorToast
+        {...props}
+        text1Style={{
```

Comment:
For text styling, Figma uses `fontSize: 16` and `fontWeight: '400'` so let's use that as well! Let's also move this styling for both the text and the toast itself onto the style sheet